app/controllers/puppetca_controller.rb

```diff
@@ -3,8 +3,7 @@ def index
     @proxy = find_proxy
     # expire cache if forced
     Rails.cache.delete("ca_#{@proxy.id}") if params[:expire_cache] == "true"
-    certs = find_certs
-    @certificates = certs.sort.paginate :page => params[:page], :per_page => Setting::General.entries_per_page
+    @certificates = find_certs
```

Comment:
This method returns only valid and pending certificates by default, then it used to be possible to select "revoked" from the dropdown (in theory) to view revoked certs too.  I can't see how to do this now?  Also, the find_certs method still uses `params[:state]` which is no longer set.